src/main/java/ArrayOperations.java

```diff
@@ -92,20 +92,18 @@ public double average() {
     /**
      * @return the average value of all elements summed up, but without the highest and the lowest value.
      */
-   // SCHAS - Zuerst sortieren und dann vorne und hinten abschneiden
 
-    public double trimmedMean() {
-        double mean = 0;
-        for (double mean2 : numbers) {
-            if (mean2 > mean || mean2 < mean) {
-                mean = mean + mean2;
-            }
-        }
+    // Ich möchte von sort() die Stellen 1-3 nehmen und damit average() ausführen. Wie stelle ich das an?
```

Comment:
Warum genau die Stellen von 1-3? 

Am einfachsten ist es wenn du die `average` Methode aufrufst und dann `minValue` und `maxValue` davon abziehst. Nicht vergessen, dass du dann 2 Werte weniger hast (für die Division).

Reply:
Hi Lukas, 
ich habe 2 Lösungsversuche. Ich versehe aber nicht, wie ich das richtig zusammenfüge - Methoden in Methoden aufrufe.  Hast einen Tipp für mich? Link,...?

Lg+
Karin

Reply:
Sorry für die späte Antwort...

Methoden können ganz einfach in anderen Methoden aufgerufen werden (wie in der Main). 
Hier in Beispiel:
Angenommen meine Methode `foo` hat als return einen int-Wert. Die Methode `bar` ist eine void-Methode.

~~~java
private void myMethod() {
 int x = foo();
 bar();
}
~~~